config/application.rb

```diff
@@ -33,6 +33,10 @@ class Application < Rails::Application
 
     config.autoload_paths << Rails.root.join('lib')
 
+    # for all non-prod environments, use the development mixpanel API
+    config.mixpanel_service_account = 'scp_terra_dev.f25a4f.mp-service-account'
```

Comment:
Just to confirm - these are not sensitive secrets?

Reply:
correct, that's just the service account name.  the correspond  secret/password is in Vault